server/services/licence/licenceStatus.ts

```diff
@@ -255,20 +257,24 @@ function getRiskManagementState(licence) {
   const checksConsideredAnswer = riskManagement?.hasConsideredChecks
   const awaitingInformationAnswer = riskManagement?.awaitingInformation || riskManagement?.awaitingOtherInformation
   const manageInTheCommunityAnswer = riskManagement?.manageInTheCommunity
-  const { proposedAddressSuitable, manageInTheCommunity } = riskManagement || {}
+  const pomConsultationAnswer = riskManagement?.pomConsultation
+  const { proposedAddressSuitable, manageInTheCommunity, pomConsultation } = riskManagement || {}
   const { bassRequested } = getBassRequestState(licence)
 
   return {
     riskManagementVersion,
     riskManagementNeeded: riskManagementAnswer === 'Yes',
     showMandatoryAddressChecksNotCompletedWarning:
-      riskManagementVersion === '2' && checksConsideredAnswer !== 'Yes' && !bassRequested,
+      (riskManagementVersion === '2' || riskManagementVersion === '3') &&
+      checksConsideredAnswer !== 'Yes' &&
+      !bassRequested,
     proposedAddressSuitable:
       proposedAddressSuitable === 'Yes' || (riskManagementVersion === '3' && manageInTheCommunity === 'Yes'),
     awaitingRiskInformation: awaitingInformationAnswer === 'Yes',
     riskManagement: getState(),
     addressUnsuitable:
       proposedAddressSuitable === 'No' || (riskManagementVersion === '3' && manageInTheCommunity === 'No'),
+    pomNotConsulted: pomConsultation === 'No',
```

Comment:
boolean for use when determining if we should show the pom consultation warning.